numpy/linalg/tests/test_linalg.py

```diff
@@ -845,6 +846,111 @@ def test_empty(self):
         assert_equal(norm(array([], dtype=self.dt)), 0.0)
         assert_equal(norm(atleast_2d(array([], dtype=self.dt))), 0.0)
 
+    def test_vector_types(self):
+        a = np.array([1, 0, 1])
+
+        exact_types = [
```

Comment:
Not a problem, the `f` kind here covers *all* the floats. I suspect you are thinking type rather than kind.